test/powershell/Modules/Microsoft.PowerShell.Security/certificateCommon.psm1

```diff
@@ -66,6 +66,46 @@ OksttXT1kXf+aez9EzDlsgQU4ck78h0WTy01zHLwSKNWK4wFFQM=
     return $certLocation
 }
 
+Function New-ProtectedCertificate
```

Comment:
We do have 2 password protected PFX in the project already

* https://github.com/PowerShell/PowerShell/blob/master/test/tools/Modules/WebListener/ClientCert.pfx
* https://github.com/PowerShell/PowerShell/blob/master/test/tools/Modules/WebListener/ServerCert.pfx

Maybe they could just be copied to the test drive or accessed directly?

Reply:
Well, but is it good to rely on WebListener's location and certs? If WebListener change in future you'll have to find what it'll break in rather unusual places like this.

Reply:
While that seems like a logical point.. that change would break the tests here and we would know about it. it's not like moving the certs would leave this tests here passing. Yes, we would have to update the tests at that time.  But WebListener is a part of the test suite so we might as well reuse what we can form it.

There is an open issue for consolidating all certs for all tests anyway as they exists in far to many places. My comment is an attempt to keep the complexity of certs in our tests at the current level without increasing it further.

Reply:
I agree that we should try to have the test certs used commonly rather than having different ones.  However, I would be fine having an issue open to refactor this as a separate PR.